compose.yaml

```diff
@@ -5,12 +5,14 @@ x-redash-service: &redash-service
     context: .
     args:
       skip_frontend_build: "true"  # set to empty string to build
+    # Enable BuildKit for faster builds with cache mounts
+    dockerfile: Dockerfile
   volumes:
     - .:/app
   env_file:
     - .env
 x-redash-environment: &redash-environment
-  REDASH_HOST: http://localhost:5001
+  REDASH_HOST: http://localhost:5000
```

Comment:
Is 5001 now the default? Restore this.

Reply:
There was some discussion about this, at this point I don't remember the details but there was a reason to switch to 5001. :)